### DISCUSSION TITLE
ANCOMBC compatibilty R versions

### DISCUSSION THREAD

**Discussion by LiiBotero** [FrederickHuangLin/ANCOMBC#43] [Q&A] [answered]
Hi! I'm trying to install this package on R studio, but the message that I get is this. "Package ‘ANCOMBC’ is not available (for R version 3.5.3) " with which versions of R its compatible?

thanks!

**Comment by FrederickHuangLin** [comment, marked as answer]
Did you install the package from Bioconductor using the following?
```
if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")

BiocManager::install("ANCOMBC")
```

If so, the package was developed after R version 4.0. You might have to update both R and Bioconductor versions.

Best,
Huang
